### ISSUE TITLE
Access multi-geo tenants using Pnp JS

### ISSUE THREAD

**Issue by Pachouuuuuu** [pnp/pnpjs#1735]
### Category
- [ ] Enhancement
- [ ] Bug
- [x] Question
- [ ] Documentation gap/issue

### Version

Please specify what version of the library you are using: [ 2.1.0 ]

Please specify what version(s) of SharePoint you are targeting: [ Online ]

### Question
I encounter similar problems as the issue #1382

We have a multi-geo tenant: https://mytenant.sharepoint.com and https://mytenanteur.sharepoint.com

I want to make a request from https://mytenant.sharepoint.com to a list contained in https://mytenanteur.sharepoint.com with the Rest API. 

As explained in the issue #1382, I tried to use the "Isolated Runtimes" object. Here is my code : 

```
sp.createIsolated<ISPConfiguration>({
        baseUrl: "https://mytenanteur.sharepoint.com/sites/contoso",
        cloneGlobal: false,
        config: {
          sp: {
            baseUrl: "https://mytenanteur.sharepoint.com/sites/contoso",
            headers: {
              Accept: "application/json;odata=verbose"
            }
          }
        },
        options: {
          credentials: "include",
          mode: "cors"
        }
      }).then((isolatedSP) => {
        isolatedSP.web().then((rootWeb) => {
          code to fetch data here ...
      });
});
```

I get the following error when my code is executed: 

```
Access to fetch at 'https://mytenanteur.sharepoint.com/sites/contoso/_api/web' from origin 'https://mytenant.sharepoint.com' has been blocked by CORS policy: 
Response to preflight request doesn't pass access control check: 
The value of the 'Access-Control-Allow-Origin' header in the response must not be the wildcard '*' when the request's credentials mode is 'include'.
```

Since I have no control over the HTTP response of the query how can I make it work?

Is it possible to achieve what I want with pnpjs? 

Should I use another authentication method instead?

Any help would be greatly appreciated. Thank you!

**Comment by patrick-rodgers** [comment]
There isn't anything we can do for CORS as it is controlled by the browser. A pattern folks use to access multiple tenants is an Azure function or other server code that provides an intermediate API. Client app calls that API, Azure function calls the tenant(s) and returns the info.

**Comment by patrick-rodgers** [comment]
Closing this issue as answered. If you have additional questions or we did not answer your question, please open a new issue, ref this issue, and provide any additional details available. Thank you!
